lib/src/actions/actions.dart

```diff
@@ -473,6 +473,21 @@ abstract class ShowMismatchesSet
   static Serializer<ShowMismatchesSet> get serializer => _$showMismatchesSetSerializer;
 }
 
+abstract class ShowDomainNameMismatchesSet
+    with BuiltJsonSerializable
+    implements Action, Built<ShowDomainNameMismatchesSet, ShowDomainNameMismatchesSetBuilder> {
+  bool get show_domain_name_mistmatches;
```

Comment:
The word "mismatches" is misspelled in the variable name here. (It's spelled "mis**t**matches".)